rootfs/etc/confd/conf.d/fluent.conf.toml

```diff
@@ -2,10 +2,20 @@
 src = "fluent.conf.tmpl"
 dest = "/fluentd/etc/fluent.conf"
 keys = [
-    "/fluentd/out/kubesys/buffer/chunk/limit",
-    "/fluentd/out/kubesys/buffer/queue/limit",
-    "/fluentd/out/kubesys/num/threads",
-    "/fluentd/out/kubeuser/buffer/chunk/limit",
-    "/fluentd/out/kubeuser/buffer/queue/limit",
-    "/fluentd/out/kubeuser/num/threads",
+    "/fluentd/out/kubeuser/buffer/overflow/action",
+    "/fluentd/out/kubeuser/buffer/chunk/limit/size",
+    "/fluentd/out/kubeuser/buffer/total/limit/size",
+    "/fluentd/out/kubeuser/buffer/flush/interval",
+    "/fluentd/out/kubeuser/buffer/retry/max/interval",
+    "/fluentd/out/kubeuser/retry/forever",
+    "/fluentd/out/kubeuser/retry/limit",
+    "/fluentd/out/kubeuser/flush/thread/count",
+    "/fluentd/out/kubesys/buffer/overflow/action",
+    "/fluentd/out/kubesys/buffer/chunk/limit/size",
+    "/fluentd/out/kubesys/buffer/total/limit/size",
+    "/fluentd/out/kubesys/buffer/flush/interval",
+    "/fluentd/out/kubesys/buffer/retry/max/interval",
+    "/fluentd/out/kubesys/retry/forever",
+    "/fluentd/out/kubesys/retry/limit",
```

Comment:
Would you prefer to name it `retry/max/times`(and _RETRY_MAX_TIMES) instead of `retry/limit`(and _RETRY_LIMIT) following [the corresponding configuration key in fluentd v0.14](https://docs.fluentd.org/v0.14/articles/buffer-section#retries-parameters)?

Reply:
Sorry for my mistake.

I'd like to corespoind confd keys with fluentd(0.14) configuration.
I fixed this in 2b3f26a